### PR TITLE
Ugrade symfony

### PR DESCRIPTION
**Pull request by lima-fox**
3.1 to 3.4

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hi2ks) [inline .idea/misc.xml:1]: you can .ignore this file

File: composer.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     },
     "require": {
         "php": ">=5.5.9",
-        "symfony/symfony": "3.1.*",
+        "symfony/symfony": "3.4",
```
Comment (hi2ks) [inline composer.json]: ^3.4

File: .idea/misc.xml (hunk 1 above)
Comment (lima-fox) [reply .idea/misc.xml:1]: ok

File: composer.json (hunk 2 above)
Comment (lima-fox) [reply composer.json]: done

**Review by hi2ks — APPROVED** [review]

**Pull request merged**